documentation/examples/prometheus-kubernetes.yml

```diff
@@ -1,22 +1,61 @@
-# A scrape configuration for running prometheus in cluster on kubernetes. It
-# will create endpoints for node and master roles, as well as any service
-# which is annotated with `prometheus_io_scrape=true`
+# A scrape configuration for running prometheus in cluster on kubernetes.
+# This uses separate scrape configs for cluster components (i.e. master, node)
+# and services to allow each to use different authentication config.
+#
+# Kubernetes labels will be added as Prometheus labels on metrics.
```

Comment:
Maybe append "...via the `label_map` relabeling action" to clarify that this doesn't happen by default.